playbooks/templates/rax-maas/ceph_rgw_stats.yaml.j2

```diff
@@ -0,0 +1,30 @@
+{% set label = "ceph_rgw_stats" %}
+{% set check_name = label+'--'+inventory_hostname %}
+{% set ceph_args = [maas_plugin_dir + "/ceph_monitoring.py", "--name", "client.raxmon", "--keyring", "/etc/ceph/ceph.client.raxmon.keyring"] %}
+{% set _ = ceph_args.extend(["rgw", "--rgw_port", "8080"]) %}
+{% set _ = ceph_args.extend(["--rgw_hosts"]) %}
+{% set _ = ceph_args.append( [ansible_host] | default([]) | join(' ')) %}
+{% set _ceph_args = ceph_args | to_yaml(width=1000) %}
+{% set ceph_args = _ceph_args %}
+
+type        : agent.plugin
+label       : "{{ check_name }}"
+period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled    : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : {{ ceph_args }}
+alarms      :
+    ceph_rgw_check.{{ ansible_host }} :
```

Comment:
Can we change `ceph_rgw_check` to `ceph_rgw_status` and remove entries for `.{{ ansible_host }}` on L19, L20, and L22?